queue_job/models/queue_job.py

```diff
@@ -506,3 +508,25 @@ def _test_job(self, failure_rate=0):
         _logger.info("Running test job.")
         if random.random() <= failure_rate:
             raise JobError("Job failed")
+
+    def _call_api(self, **kwargs):
```

Comment:
api is vague, I would rather name it `_call_webhook` to have a consistant naming